[Doc] Update global_settings.md (#6370)
taichi-dev/taichi: docs/lang/articles/reference/global_settings.md

```

:::

## Going high precision

> The function `ti.sqrt` has a lower precision on the CUDA backend than on the CPU backend, see this [issue](https://github.com/taichi-dev/taichi/issues/6346). To work around this problem, you can set the default precision to `ti.f64` and turn off the `fast_math` option:

Reviewer: ```suggestion
The function `ti.sqrt` (and those who rely it, such as `ti.norm`) may have a lower precision on the CUDA backend than on the CPU backends, see this [issue](https://github.com/taichi-dev/taichi/issues/6346). To work around this problem, you can set the default precision to `ti.f64` and turn off the `fast_math` option:
```

> The function `ti.sqrt` (and those who rely on it, such as `ti.norm`) may have a lower precision on the CUDA backend than on the CPU backends, see this [issue](https://github.com/taichi-dev/taichi/issues/6346). To work around this problem, you can set the default precision to `ti.f64` and turn off the `fast_math` option:

Reviewer 2: ```suggestion
The function `ti.sqrt` (and those that rely on it, such as `ti.norm`) may have a lower precision on the CUDA backend than on the CPU backends. See this [issue](https://github.com/taichi-dev/taichi/issues/6346) for more information. To work around this problem, set the default precision to `ti.f64` and turn off the `fast_math` option:
```
